### PR TITLE
55786/feat​/add new investigation status

### PR DESCRIPTION
**Pull request by ornahasson**

### CODE REVIEW

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+INSERT INTO investigation_status (display_name, id)
+SELECT 'לא נחקר',100000005
+WHERE
```
Comment (chenFriedman) [inline server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:3]: the id is not unique and then lunes 3-6 are unnecessary?

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    NOT EXISTS (
+        SELECT id FROM investigation_status WHERE id = '100000005'
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
+SELECT 'מדיניות משרד הבריאות',100000008,100000005
+WHERE
```
Comment (chenFriedman) [inline server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:10]: same

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    NOT EXISTS (
+        SELECT id FROM investigation_sub_status WHERE id = 100000008
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
```
Comment (chenFriedman) [inline server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:15]: I'm not sure i understand whats your trying to do here

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+SELECT 'מדיניות לשכה',100000009,100000005
+WHERE
+    NOT EXISTS (
+        SELECT id FROM investigation_sub_status WHERE id = 100000009
+    );
+
+INSERT INTO investigation_sub_status (display_name, id, parent_status)
```
Comment (chenFriedman) [inline server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:22]: I'm not sure i understand whats your trying to do here

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql (hunk 1 above)
Comment (barnahum8) [reply server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:3]: @ornahasson read about ON CONFLICT:
https://www.postgresql.org/docs/9.5/sql-insert.html

File: server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql (hunk 4 above)
Comment (barnahum8) [reply server/src/ScriptRunner/Scripts/index/add_not_investigated_to__investigation_statuses.sql:22]: @ornahasson read about ON CONFLICT:
https://www.postgresql.org/docs/9.5/sql-insert.html

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**